Components/avatar.css

```diff
@@ -0,0 +1,29 @@
+.avatar img{
+    border-radius: 50%;
+    object-fit: cover;
+}
+
+.avatar-xx-l img{
```

Comment:
just to re-iterate what Mehul has already said, whenever dealing with sizing use words like: 
'xxs', 'xs', 'sm', 'md', 'lg', 'xl', '2xl' and so on...